providers/src/airflow/providers/apache/drill/CHANGELOG.rst

```diff
@@ -27,6 +27,20 @@ Changelog
 ---------
 
 
+2.8.1
```

Comment:
I wonder (not strong though) if it should be 2.9.0 maybe? We added support for semicolon stripping for all affected sql providers ?

Reply:
Right!